src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/SwitchPartitionsOutAllTables.sql

```diff
@@ -1,7 +1,6 @@
 ﻿--IF object_id('SwitchPartitionsOutAllTables') IS NOT NULL DROP PROCEDURE dbo.SwitchPartitionsOutAllTables
 GO
 CREATE PROCEDURE dbo.SwitchPartitionsOutAllTables @RebuildClustered bit
-WITH EXECUTE AS SELF
```

Comment:
Why is dbo not needed here?

Reply:
Permissions from one stored proc to another are transferable. I don't think impersonation is needed.